.github/workflows/manual-sync.yml

```diff
@@ -0,0 +1,81 @@
+name: Sync from osu! upstream
+
+# Workflow dispatch can take inputs like any other github.event
+# As part of #5090 - we should evaluate adding inputs and outputs to refine this.
+on:
+  workflow_dispatch:
+    inputs:
+      force:
+        description: "Overwrite any changes in the target repository? (true/false)"
+        required: true
+        default: "false"
+
+jobs:
+  sync:
+    name: Execute Sync Task
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure Git to use your name
+        shell: bash
+        run: |
+          set -eo pipefail
+
+          #FIXME: We're using the noreply email of GitHub for now. Let's figure out a way to get the committer's email.
+          git config --global user.name ${{ github.actor }}
+          git config --global user.email ${{ github.actor }}@users.noreply.github.com
+
+
+      - name: Begin Pull Task
+        shell: bash
+        run: |
+         set -eo pipefail
+
+         # sanitize github input into boolean
+         # https://stackoverflow.com/questions/14700579/bash-converting-string-to-boolean-variable
+         function boolean () {
+            case $1 in
+              true) echo true ;;
+              false) echo false ;;
+              *) echo "Unknown operand \"$1\"" 1>&2; exit 1 ;;
+            esac
+         }
+
+         #Upstream repository URL, this is where everything ties down.
+         UPSTREAM_REPO="https://github.com/ppy/osu-wiki"
+         BACKUP_BRANCH_NAME="backup-$(git branch --show-current)"
+         BRANCH_NAME="$(git branch --show-current)"
+
+         git remote add upstream $UPSTREAM_REPO
+         git fetch upstream
+
+         if [ ! $(boolean ${{ github.event.inputs.force }}) ]; then
+           # Obviously this is pretty dangerous, this needs to be properly fixed later on.
```

Comment:
why not fix it now?

Reply:
Oh this should be fixed now (I hope), forgot to remove this.